lib/chrome.js

```diff
@@ -58,6 +59,16 @@ class Events {
 
 const MIN_INTERVAL = 1000;
 
+function keyToId(key) {
+  let md5Str = md5(key);
+  const parts = [];
+  [8,4,4,4,12].map(numChars => {
```

Comment:
afaict `node-uuid` generates random uuid's, and there's no way to use it just to add the hyphens into an md5 string, or generate a uuid deterministically starting from a given string like `key`?